ofl/encodesans/METADATA.pb

```diff
@@ -10,81 +10,15 @@ fonts {
   filename: "EncodeSans-Thin.ttf"
   post_script_name: "EncodeSans-Thin"
   full_name: "Encode Sans Thin"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 200
-  filename: "EncodeSans-ExtraLight.ttf"
-  post_script_name: "EncodeSans-ExtraLight"
-  full_name: "Encode Sans ExtraLight"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 300
-  filename: "EncodeSans-Light.ttf"
-  post_script_name: "EncodeSans-Light"
-  full_name: "Encode Sans Light"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 400
-  filename: "EncodeSans-Regular.ttf"
-  post_script_name: "EncodeSans-Regular"
-  full_name: "Encode Sans Regular"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 500
-  filename: "EncodeSans-Medium.ttf"
-  post_script_name: "EncodeSans-Medium"
-  full_name: "Encode Sans Medium"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 600
-  filename: "EncodeSans-SemiBold.ttf"
-  post_script_name: "EncodeSans-SemiBold"
-  full_name: "Encode Sans SemiBold"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 700
-  filename: "EncodeSans-Bold.ttf"
-  post_script_name: "EncodeSans-Bold"
-  full_name: "Encode Sans Bold"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 800
-  filename: "EncodeSans-ExtraBold.ttf"
-  post_script_name: "EncodeSans-ExtraBold"
-  full_name: "Encode Sans ExtraBold"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
-}
-fonts {
-  name: "Encode Sans"
-  style: "normal"
-  weight: 900
-  filename: "EncodeSans-Black.ttf"
-  post_script_name: "EncodeSans-Black"
-  full_name: "Encode Sans Black"
-  copyright: "Copyright 2012 The Encode Project Authors (impallari@gmail.com), with Reserved Font Name \"Encode Sans\"."
+  copyright: "Copyright 2018 The Encode Project Authors (https://github.com/thundernixon/Encode-Sans), with Reserved Font Name \'Encode Sans\'."
 }
 subsets: "latin"
 subsets: "latin-ext"
 subsets: "menu"
 subsets: "vietnamese"
+axes {
+  tag: "wght"
+  min_value: 100.0
+  default_value: 100.0
```

Comment:
I believe this should be 400, since CSS requires the 400 Regular to be the default.

Reply:
Thanks for the sanity check! I do think this is correct...

As far as I know, a variable font must have a default axis values that align with its default instance. 

At the very least, this seems to be a requirement to build with FontMake. If I set a default of `400` for `wght` in the designspace, then try to generate a variable font, it fails.

This:

```
<axis tag="wght" name="Weight" minimum="100" maximum="900" default="400">
```

...results in this:

```
$ fontmake -o variable -m master_ufo/EncodeSans.designspace
 
# build information
 
File "/Users/stephennixon/type-repos/google-font-repos/Encode-Sans/venv/lib/python3.7/site-packages/fontTools/varLib/__init__.py", line 748, in load_designspace
    assert base_idx is not None, "Base master not found; no master at default location?"
AssertionError: Base master not found; no master at default location?
```

I assume that the default instance must be described for legacy apps that can't handle variable fonts. In the past, we have had conversations around whether this should be the light or bold master, if there is no middle master to provide the default values.

I made a quick test of this for browsers, just to check whether a default `wght` value of 100 would cause any issues. As far as I can tell, it doesn't.

Chrome:
![image](https://user-images.githubusercontent.com/7355414/56532005-02fe6d00-6523-11e9-9153-8e6aa25845f5.png)

---

Safari:
![image](https://user-images.githubusercontent.com/7355414/56532122-3640fc00-6523-11e9-99a9-e97869612961.png)

---

Firefox:
![image](https://user-images.githubusercontent.com/7355414/56532071-232e2c00-6523-11e9-8eab-d5200bd62a48.png)

---

Windows 10, Edge:
![image](https://user-images.githubusercontent.com/7355414/56532454-c7b06e00-6523-11e9-8264-2c1f7db58787.png)

Reply:
I _think_ the MS opentype spec also stipulates this must be the case, though their definition is admittedly somewhat complex. I take this to mean that the default value of an instance should represent what is in the glyph table, before any deltas are applied:

> The default value interacts with the glyph and glyph variations tables in a particular way: the variation instance that has the default coordinate value for each axis will use glyph outlines as defined in the glyph table, without any variations from the glyph variations table applied. This instance is referred to as the default instance.

[from the fvar spec](https://docs.microsoft.com/en-us/typography/opentype/spec/fvar#variationaxisrecord)

Reply:
Something I was curious about that I hadn't tested earlier ... if the `font-weight: 100 900;` is not declared in the `@font-face`, Encode Sans still shows up as a `400` weight in Chrome, while all weights just continue to show up as before in Safari.

![image](https://user-images.githubusercontent.com/7355414/56548957-7b7b2300-654f-11e9-820f-5260a0289c4b.png)